magma/passes/terminate_unused.py

```diff
@@ -2,12 +2,20 @@
 from ..is_definition import isdefinition
 
 
+def _terminate_if_unwired_output(port):
+    if port.is_mixed():
+        # Sum so it doesn't short circuit
+        return sum(_terminate_if_unwired_output(p) for p in port)
```

Comment:
```suggestion
        terminated = (_terminate_if_unwired_output(p) for p in port)
        return any(terminated)
```

This is a little more readable